### PR TITLE
chore: homepage content nits

### PR DESCRIPTION
**Pull request by gariasf**
I thought the Brisa website could have some improvements. This PR is a proposal to make it a bit more appealing, and consistent.

Happy to hear your feedback!

### CODE REVIEW

**Comment by gariasf** [comment]
I would love to also make some more design changes, but I will probably do that later.

**Comment by aralroca** [comment]
@gariasf The initiative is good, but for the moment I don't approve of it, there are several things that don't convince me.

I recommend that you divide the PR, the spelling mistakes without changing styles on one side and the styles on the other. For the moment my disapproval is in the styles.

Screenshot 2024-10-08 at 14 52 09

- It seems to me good to change the emojis from unicode to SVG, but they would have to be well integrated for all the devices and I would add the icons of the sections below.

Screenshot 2024-10-08 at 14 52 53

- There shouldn't be a case where it doesn't do a wrap but the order is the wrong way round, you will probably have to change the media query. Now the 2 sections with this screen size have a different order than before. Besides, the icon looks very small. It needs to be integrated.
- About the font family, I am not convinced to change it. I had a hard time finding one that was easy to read, we were using https://fonts.google.com/specimen/Inter

Screenshot 2024-10-08 at 14 54 09

- why do we delete the image?

Screenshot 2024-10-08 at 14 54 23

- why did we remove the padding on the video caption?

Screenshot 2024-10-08 at 14 54 38

- I saw it more attractive with the emojis.... 😕
- nit: Brisa is not a "projec" is a "project" 😉

These are all my thoughts on seeing it. Nevertheless, the initiative is good, and thanks for all the input. Perhaps here it would be good for others to give their opinion of the changes, CC:  @danielart  @AlbertSabate @adriablancafort

File: packages/www/src/pages/index.tsx
Comment (aralroca) [inline packages/www/src/pages/index.tsx]:
```suggestion
              Brisa is an open-source project, and is backed by contributions
```

**Comment by gariasf** [comment]
Thanks for the feedback, @aralroca. In my opinion, I wouldn't use icons at all, and I find emojis unprofessional. Moreover, the emojis, being so colourful and with the same opacity as the title, take away attention, when I'd say the focus should actually be on the text. 
The reason I decided to remove the cross-platform image is that it looked improper and too complex on the page, especially on the desktop.  Perhaps a more minimalistic illustration would benefit that section, in line with the rest of the website, which is quite minimalistic. Moreover, it felt a bit off that that section was the only one to have an image. (Also, there is no Windows device in it.)

If you look at the overall distribution, it feels a bit unbalanced, there is quite some blank space in the other sections, especially on the i18n one.
image

I'd like to leave some extra design thoughts that I would love to address later:
I still feel like the line height is too large, it's harder to read for me that way. Especially the code.
image

The section titles are not aligned with the code boxes.
image

All media has a border radius except your image at the bottom.

The sections, including the navigation, have low contrast between them, that makes it difficult to see when one starts and the other ends.
image

I will probably take all of this and move it to PR, then we can discuss there.

You're right, though, I overstepped a bit and I didn't spend the time to check the responsiveness, I got a bit excited. I've fixed it, only nits now. And amazing job on Brisa, by the way!

File: packages/www/src/pages/index.tsx
Comment (AlbertSabate) [inline packages/www/src/pages/index.tsx]: Should be single quote. Make sure to install biome in your IDE for keeping file consistency. This will be positive for all contributors keeping the same coding style. We, so far, are working towards respect biome standards.

File: packages/www/src/pages/index.tsx
Comment (AlbertSabate) [inline packages/www/src/pages/index.tsx]:
```suggestion
              In Brisa everything runs on the server by default, except the{" "}
```

File: packages/www/src/pages/index.tsx
Comment (AlbertSabate) [inline packages/www/src/pages/index.tsx]: On this one we need a bit more clarity. What the original is saying is that Brisa RPC has some js size, so client will have some Kb of JS in order to handle Brisa RPC connection.

File: packages/www/src/pages/index.tsx
Comment (AlbertSabate) [inline packages/www/src/pages/index.tsx]:
```suggestion
              Brisa has built-in internationalization (i18n) support that allows you to{' '}
```

File: packages/www/src/pages/index.tsx
Comment (AlbertSabate) [inline packages/www/src/pages/index.tsx]:
```suggestion
              <b>translate</b> your pages and <b>routes</b>, while loading
```

File: packages/www/src/pages/index.tsx
Comment (AlbertSabate) [inline packages/www/src/pages/index.tsx]: Brisa implements Tauri. On this one I like more the original sentence.

File: packages/www/src/pages/index.tsx
Comment (AlbertSabate) [inline packages/www/src/pages/index.tsx:235]: Mission of a product should be defined by the original author. On this one I think we should respect Aral's original wording. Unless you want to suggest a change on the mission, I would prefer to respect the wording. This is my personal thinking, happy to listen other arguments.

**Review by AlbertSabate — COMMENTED** [review]
Some thoughts I have after reviewing the PR. Thanks for helping us with the copy, sometimes its hard to find the right wording :)

**Comment by AlbertSabate** [comment]
Reviewed the comments about the emojis. And I can give my thoughts on that. In my opinion emojis are not unprofessional when used with moderation, I think it gives a young and friendly vibe to the project.

I also asked Chat GPT for opinion, and this is what it told me:

Screenshot 2024-10-08 at 11 58 58 PM
Screenshot 2024-10-09 at 12 09 36 AM

In regards of the spaces on the landing, I agree it could be better padding and margins needs more work. Contrasts I'm not a designer, can't help here :)

File: packages/www/src/pages/index.tsx
Comment (gariasf) [reply packages/www/src/pages/index.tsx]: My bad! I had my usual settings set up. I suggest we add information about it to the Contributing Guide, as it doesn't say anything regarding the coding pipeline, code style or linting process. It also doesn't say anything regarding commit style either, we might want to have a GitHub workflow that runs on PRs and checks that their title conforms to conventional commits. Another option is to use husky or any other git hook tooling to help with this. Will fix the format as soon as I can.

File: packages/www/src/pages/index.tsx
Comment (gariasf) [reply packages/www/src/pages/index.tsx]: Why do you think we need to say the same thing again? Isn't i18n just an abbreviation of internationalization?

File: packages/www/src/pages/index.tsx
Comment (gariasf) [reply packages/www/src/pages/index.tsx]: I had my debate regarding this one, will loading convey the meaning we want here? It seems like loading means more things than just getting something over the wire. This is why I went with downloading, but understand the difference is negligible. Perhaps someone else has more thoughts about it.

File: packages/www/src/pages/index.tsx
Comment (gariasf) [reply packages/www/src/pages/index.tsx]: I might be missing something here, then. Because as far as I know, you use and build React, JavaScript, or Angular with Electron or Tauri. I wouldn't say they “implement” them, unless there is some sort of API support, in which case it's more of an integration. I am a bit ignorant on this particular topic, so I would love to better understand how Brisa works with Tauri, @aralroca can you help? 
The way I see it, if Brisa can be built to be embedded with Tauri, then I would say that it can be used with Tauri, or is compatible with Tauri.

File: packages/www/src/pages/index.tsx
Comment (gariasf) [reply packages/www/src/pages/index.tsx:235]: I'll let Aral decide on this one. @aralroca , WDYT?

**Comment by gariasf** [comment]
In general, I think we should be mindful about the impression that is put out there regarding Brisa. In the end, what matters is the tool itself, that's true. With that said, a landing page is also marketing and first impressions matter.

File: packages/www/src/pages/index.tsx
Comment (AlbertSabate) [reply packages/www/src/pages/index.tsx]: > I suggest we add information about it to the Contributing Guide, as it doesn't say anything regarding the coding pipeline, code style or linting process. It also doesn't say anything regarding commit style either, we might want to have a GitHub workflow that runs on PRs and checks that their title conforms to conventional commits.

Total agree with this. I'll open an issue to work this out. (In another PR).

File: packages/www/src/pages/index.tsx
Comment (AlbertSabate) [reply packages/www/src/pages/index.tsx]: This mainly for clarity, I understand your point as well, and I agree. Maybe we could do a Reference page or footer and list there all abbreviations?

The point of having it explained somewhere is because some people won't know the naming convention. Because they decide to learn coding with Brisa, because they didn't have to use it before, etc. And we do not have it unified enough, some places we call it i18n and other internationalization. It is for that the reinforcement on this one.

But maybe good idea to go with your suggestion here and write in another place the definition / references.

File: packages/www/src/pages/index.tsx
Comment (aralroca) [reply packages/www/src/pages/index.tsx]: Tauri is almost transparent for the developer in Brisa, this means that only needs to modify one configuration property in `brisa.config.ts` file to decide if build an .apk, .ica, .exe, .dmg, etc. In this way facilitate transforming your web app to an mobile/desktop app in a easy way. So, our CLI configures and integrates Tauri to do this, but is a Brisa feature.

File: packages/www/src/pages/index.tsx
Comment (aralroca) [reply packages/www/src/pages/index.tsx:235]: It's the same mission but simplified, LGTM.

**Comment by gariasf** [comment]
I'll leave it with just my rephrasing nits. PR is ready.

**Review by aralroca — CHANGES_REQUESTED** [review]
Thanks a lot Guillem!  🔥 Everything is fine, the only part I would change is:

> “Brisa can also be used with Tauri”.

Because Tauri is transparent and you don't need to install it manually, and it is a feature of Brisa that underneath uses the integration with Tauri. It was better explained before, this leads to confusion, because in theory Tauri could be used in any web framework, the grace is that in Brisa is fully integrated by default.

File: packages/www/src/pages/index.tsx
Comment (aralroca) [reply packages/www/src/pages/index.tsx]: @gariasf after this change we can merge the PR

**Review by aralroca — APPROVED** [review]

**Comment by aralroca** [comment]
Thanks for that contribution @gariasf

**Pull request merged**
